src/com/rental/utils/XmlWorker.java

```diff
@@ -118,6 +147,12 @@ public void returnProductOnStock(int userInputSelected) {
         saveChangesInXmlProductsFiles();
     }
 
+    /**
+     * Changes the name of the attribute in the users.xml file to null.
+     * Aditionally, it checks and compares the user input and xml file if the value can be added.
+     * In the case of a positive implementation, the data is entered into the appropriate xml file.
+     * @param login reference to the object created in the RentalProcessor class
+     */
     public void deleteProductFromUserBase(Login login) {
```

Comment:
Read about the CRUD operations, what(arguments) they should receive, what they should return.